packages/Liquid-Tests.package/LQStatisticsWorkbenchIntegrationTests.class/instance/testStatisticsWorkbenchIsDefined.st

```diff
@@ -0,0 +1,6 @@
+as yet unclassified
```

Comment:
[sand] unclassified. Maybe we should run the linter on Liquid-Tests as well to prevent stuff like this? That would require some exceptions (e.g. LiquidLintTests has too many methods)

Reply:
Hmm, I'm not sure. There are pros and cons to that decision for sure. You know best about the Linter, how much work would that change be?

Reply:
I'll try that somewhere else. Consider this discussion solved :)